public/app/features/plugins/extensions/getPluginExtensions.test.tsx

```diff
@@ -611,7 +611,7 @@ describe('getObservablePluginExtensions()', () => {
     });
   });
 
-it('should emit the initial state when no changes are made to the registries', async () => {
+  it('should emit the initial state when no changes are made to the registries', async () => {
     const observable = getObservablePluginExtensions({ extensionPointId }).pipe(first());
```

Comment:
You should be able to remove all `.pipe(first())` and `pipe(take(2)` calls and the tests should pass?

Reply:
Hmm, yeah, I tried, but then unfortunately it said that the observable didn't complete. 🤔

Reply:
You're correct the `ReplaySubject` in the registry is never complete so that's why, good catch 🙌  Sorry for the confusion 🙈

Reply:
No worries mate, I was quite confused about it, too :)